product_docs/docs/transporter/ba_preview/getting_started/installing/index.mdx

```diff
@@ -6,32 +6,28 @@ navigation:
   - linux_x86_64
 ---
 
-To install EDB Transporter for internal purposes:
+Select a link to access the applicable installation instructions.
 
-### Accessing CloudSmith
+## Linux [x86-64 (amd64)](linux_x86_64)
 
-Log in to Okta, and navigate to the CloudSmith tile to access the dev CloudSmith repository. 
+### Red Hat Enterprise Linux (RHEL) and derivatives
 
-### Finding the appropriate package
+- [RHEL 8](transporter_rhel_8), [RHEL 7](transporter_rhel_7)
 
-Look for the last `cdcreader` version for the architecture and operating system you are using.
+- [Oracle Linux (OL) 8](transporter_rhel_8), [Oracle Linux (OL) 7](transporter_rhel_7)
 
-### Installing the package
+- [Rocky Linux 8](transporter_other_linux_8)
 
-Navigate to the setup page of the selected `cdcreader` and follow the installation instructions.
+- [AlmaLinux 8](transporter_other_linux_8)
 
-### Verifying the installation
+- [CentOS 7](transporter_centos_7)
```

Comment:
are the above links correct? should they all be prefixed with linux_x86_64/ ?

Reply:
You're right. I fixed this issue in the recent commit.